### PR TITLE
[Hotfix :fire: ] - missing wallet name issue fix

### LINKED ISSUE #1929: [1.5.0] Prod: no wallet name in the Activity modal
1. Connect any wallet using WC with the app
2. Open activity modal

**AR:** 'Connected with WalletConnect' is displayed

**ER**: the connected wallet should be mentioned in the Activity modal

### PR DESCRIPTION
**Pull request by nenadV91**
# Summary

Fixes #1929

This will fix the issue with missing wallet name in the Activity Modal such as 

Screenshot from 2021-11-26 16-07-04
Screenshot from 2021-11-26 16-12-15

### CODE REVIEW

**Review by alfetopito — APPROVED** [review]

**Comment by alfetopito** [comment]
Actually, I think @anxolin mentioned in another issue.

Create a hotfix branch first from master, then point the PR to that branch:

`git checkout master && git pull && git checkout -b hotfix/1.5.1`

**Comment by elena-zh** [comment]
Hey @nenadV91 , changes LGTM!
However, I noticed, that a list of wallets that are currently supported in the Prod, has become not supported in this PR:
 - 1Inch
 - DeFi wallet
 - WallETH
Screenshot_1
image

File: src/custom/hooks/useWalletInfo.ts [hunk 1]
```diff
@@ -65,7 +65,7 @@ function checkIsSupportedWallet(params: {
 async function getWcPeerMetadata(connector: WalletConnectConnector): Promise<{ walletName?: string; icon?: string }> {
   const provider = (await connector.getProvider()) as WalletConnectProvider
 
-  const meta = provider.walletMeta
+  const meta = provider.signer.connection.wc.peerMeta
```
Comment (W3stside) [inline src/custom/hooks/useWalletInfo.ts:68]: i think you need to create an `||`  check here as the conditional on line `69` is failing and causing wallets to not appear.

`const meta = provider.walletMeta || provider.signer.connection.wc.peerMeta`

**Review by W3stside — CHANGES_REQUESTED** [review]
|| conditional check

**Review by elena-zh — APPROVED** [review]
Hey @nenadV91 , based on the meeting discussion, we should not support mentioned above wallets until we resolve all related integration issues with them. 
So the changes under this PR look good to me.

**Review by anxolin — APPROVED** [review]
> However, I noticed, that a list of wallets that are currently supported in the Prod, has become not supported in this PR:
> 1Inch
> DeFi wallet
> WallETH


Those are supposed to be unsupported. So this is correct in this PR and bad in prod. This is why I mentioned the bug in production was bad, it probably affects gnosis safe too this is way we are deploying the hotfix 1st time in the morning and then we do a second hotfix with this.

**Comment by anxolin** [comment]
Nenad, as leandro pointed out, this PR sould point to a hotfix. Not master

**Comment by nenadV91** [comment]
@anxolin It was but then we merged hotfix/1.5.1 so this happend `Base automatically changed from hotfix/1.5.1 to master 2 hours ago` so I will create another hotfix/1.5.2 branch and point it to that one.

File: src/custom/hooks/useWalletInfo.ts (hunk 1 above)
Comment (nenadV91) [reply src/custom/hooks/useWalletInfo.ts]: Done

**Comment by anxolin** [comment]
So, for me good to go, just double check with David about his comment just in case

**Comment by elena-zh** [comment]
This issue was reverted by 1.5.1 Hotfix. 
So, I think, if we deploy changes in 1.5.2 to Prod, we will return back #1930 issue that has not been fixed yet...

**Pull request merged**